src/olympia/promoted/models.py

```diff
@@ -179,13 +180,6 @@ def update_es_for_promoted(sender, instance, **kw):
     update_search_index(sender=sender, instance=instance.addon, **kw)
 
 
-@receiver(
-    models.signals.post_save, sender=PromotedAddon, dispatch_uid='promoted_prereview'
-)
-def update_due_date_for_pre_review(sender, instance, **kw):
```

Comment:
No longer needed since belonging to a `PromotedAddon` no longer directly cause something to have a due date (it has to go through `auto_approve` first).